Libbna/CUSTOM-CMS
# #71: Fixed the validation checks and also cleaned up some code

Steps to test:
* Check all the forms on the site(Registration, login, article, block, menu, etc).
* Try submitting entire form empty
* Try submitting the form with an empty field

Expected behaviour
* The user should get redirected to error page with appropriate message for the user
